src/lib/fileUtilities.ts

```diff
@@ -1,7 +1,13 @@
-import { promisify } from "util";
+import fs from "fs";
 import glob from "glob";
+import mkdirp from "mkdirp";
+import path from "path";
+import { promisify } from "util";
 
 const globAsync = promisify(glob);
+const asyncReadFile = promisify(fs.readFile);
+const asyncWriteFile = promisify(fs.writeFile);
+const mkdirpAsyn = promisify(mkdirp);
```

Comment:
[minor] Typo here `Async`?

Reply:
Might also consider making the `async` part consistently at the beginning or end

Reply:
Ooooops.  Done!